docs/cli-help.txt

```diff
@@ -22,14 +22,11 @@ optional arguments:
   -v, --verbose         This flag can be used multiple times to incrementally
                         increase the verbosity of output, e.g. -vvv. (default:
                         -2)
-  -t, --text            Use the text output instead of the curses UI.
```

Comment:
Ok it seems the correct choice here is to not generate the documentation on that website based on master, but instead to generate it based on the release branch. Is there already an issue for that, or a reason why we don't already do that?